### ISSUE TITLE
about seq5 in synthetic dataset

### ISSUE THREAD

**Issue by edwardzcl** [aa-samad/conv_snn#12]
In you paper, the seq5 example in Figure 4 is rotated with a random degree between 0 and 360 degrees. but it can't be distinguished between clockwise or counterclockwise. Because i can't find some evidence for rotation direction. In other words,  clockwise 30 degree is equivalent to counterclockwise 270 degree, so how you can classify the rotation direction.

**Comment by aa-samad** [comment]
Ok maybe We did not explain this correctly, 
The rotation is like this (As it is the code):
theta = 0    ->  each step:  theta = theta + randint(0, 360, 36)  and the image is rotated with theta with respect to original image.
Therefore, if the synthetic dataset generator gets randint > 180 for most of the frames that way the rotation becomes counter clock-wise, which is possible. But, this is the extreme scenario, In most cases we limit random generation between 0 and 180 or generator has non-uniform distribution and mean close to 0. 
In conclusion, If we plot the probability of rotations, it is mostly clockwise for setup (not all of them, in order to increase the challenge).

Notes:
1. I believe in the experiments the setup was 0 to 270, with normal distribution and mean of 135. which meant some counter-clockwise rotations occured during clockwise rotation (to have noise). This should be updated in the papers text, Thankyou for mentioning it.
2. clockwise 90 = counterclockwise 270

**Comment by edwardzcl** [comment]
Ok, thanks for your explanation.
Yes, there is a mistake in my description, i.e. clockwise 90 = counterclockwise 270.
So, compared with seq2, you just add noise or random rotation in seq5 to increase the challenge?

**Comment by aa-samad** [comment]
Correct.
The reason behind it is that: usually the conv3D can pickup regular temporal pattern pretty quickly
